main/sw/inc/shellio.hxx

```diff
@@ -367,32 +363,32 @@ public:
     void   SetBaseURL( const String& rURL );
 
 	sal_Bool   IsOld() const;
-	sal_uLong  ConvertToNew();				// Textbausteine konvertieren
+	sal_uLong  ConvertToNew();				// convert AutoText entries (ger: Textbausteine)
 
-	sal_uInt16 GetCount() const;						// Anzahl Textbausteine ermitteln
-	sal_uInt16 GetIndex( const String& ) const;			// Index fuer Kurznamen ermitteln
+	sal_uInt16 GetCount() const;						// return amount of AutoText entries
+	sal_uInt16 GetIndex( const String& ) const;			// return Index of shortnames
 	sal_uInt16 GetLongIndex( const String& ) const;		//Index fuer Langnamen ermitteln
```

Comment:
Maybe "Return index of longnames"?

Reply:
Watzlav has pointed this out, too. I have to work his comments in. Did not have the time. At the end of the week I think I can put again some more time.

Reply:
done